aloe_django/steps/models.py

```diff
@@ -292,25 +275,25 @@ def test_existence(model_or_queryset, data):
     return False
 
 
-def models_exist(model, data, queryset=None,
-                 existence_check=None,
-                 should_exist=True):
+def _model_exists_step(step, model, should_exist):
     """
-    Check whether the models defined by @data exist in the @queryset.
+    Tests for the existance of a model matching the given data.
```

Comment:
Also, please use active voice in descriptions for consistency ("Test stuff", not "Tests stuff").